hetio/neo4j.py

```diff
@@ -287,6 +287,29 @@ def construct_unique_nodes_clause(metarels, unique_nodes):
 
     return unique_nodes_query
 
+def create_path_return_clause(path_style='list'):
+    """
+    Create a Cypher query clause to return paths either as a list or as a string.
+    As formatting the output as a string is less efficient in terms of database
+    hits, 'list' is the default style
+
+    Parameters
+    ----------
+    path_style : str
+        the way the user wants the path returned. Currently supported options
+        are 'list' and 'string'
+    """
+    if path_style == 'string':
+        return "substring(reduce(s = '', node IN nodes(path)| s + '–' + node.name), 1) AS path,"
+    elif path_style == 'list':
+        return "extract(n in nodes(path) | n.name) AS path,"
+    else:
+        err_string = str(path_style) + (" is not a style currently implemented by "
```

Comment:
Preferable to use `"{}".format` syntax. Cannot use f-literals yet since we still support python 3.4 and 3.5 although perhaps we should drop this support soon.